keras/layers/normalization.py

```diff
@@ -139,7 +139,7 @@ def call(self, x, mask=None):
         elif self.mode == 1:
             # sample-wise normalization
             m = K.mean(x, axis=-1, keepdims=True)
-            std = K.std(x, axis=-1, keepdims=True)
+            std = K.sqrt(K.var(x, axis=-1, keepdims=True) + self.epsilon)
             x_normed = (x - m) / (std + self.epsilon)
```

Comment:
Is that OK that self.epsilon is added twice: once while calculating std thus preventing is from being zero and then later when normalizing x?